### PR TITLE
ZKUI-254 Improve Replication Workflow

### PR DESCRIPTION
**Pull request by ghivert**

### CODE REVIEW

File: src/react/workflow/ReplicationForm.tsx [hunk 1]
```diff
@@ -179,12 +187,11 @@ function ReplicationForm({
...
-                  render={({ field: { onChange, value: sourceBucket } }) => {
-                    const options = sourceBucketOptions(
-                      replicationsQuery.data || [],
-                      bucketList,
-                      locations,
-                    );
+                  render={({
+                    field: { onChange, onBlur, value: sourceBucket },
+                  }) => {
+                    // const streams = replicationsQuery.data || [];
```
Comment (JBWatenbergScality) [inline src/react/workflow/ReplicationForm.tsx]: Commented code ?

File: src/react/workflow/ReplicationForm.tsx [hunk 2]
```diff
@@ -304,11 +327,15 @@ const RenderDestination = ({
...
           name={name}
-          render={({ field: { onChange, value: destinationLocations } }) => {
+          render={({
+            field: { onChange, onBlur, value: destinationLocations },
+          }) => {
             if (!Array.isArray(destinationLocations)) return null;
             return destinationLocations.map((destLoc: string, index) => {
               const options = destinationOptions(locations);
-              const err = errors[`${prefix}destinationLocation.${index}`];
+              const prefix_ = `${prefix}destinationLocation.${index}`;
```
Comment (JBWatenbergScality) [inline src/react/workflow/ReplicationForm.tsx]: nit: is `prefix_` a specific nomenclature you want to use here or a fat-finger typo ?

File: src/react/workflow/ReplicationForm.tsx (hunk 1 above)
Comment (ghivert) [reply src/react/workflow/ReplicationForm.tsx]: Oops, my mistake

File: src/react/workflow/ReplicationForm.tsx (hunk 2 above)
Comment (ghivert) [reply src/react/workflow/ReplicationForm.tsx]: Was a specific nomenclature, changed name. (It's a standard way of doing in Haskell / PureScript / Elm world, an old habit 😅)

**Review by JBWatenbergScality — APPROVED** [review]

**Pull request merged**